web/src/server/api/routers/generations/getAllQuery.ts

```diff
@@ -10,28 +10,36 @@ import { getAllGenerationsSqlQuery } from "@/src/server/api/routers/generations/
 const getAllGenerationsInput = GenerationTableOptions.extend({
   ...paginationZod,
 });
+
+export type ScoreSimplified = {
+  name: string;
+  value: number;
+  comment?: string | null;
+};
+
 export type GetAllGenerationsInput = z.infer<typeof getAllGenerationsInput>;
 
+export type ObservationViewWithScores = ObservationView & {
```

Comment:
For completeness, transformStreamToJsonLines should also take this type but only use the data it is currently using.